static/js/checks.js

```diff
@@ -305,4 +311,75 @@ $(function() {
         var text = e.trigger.getAttribute("data-clipboard-text");
         prompt("Press Ctrl+C to select:", text);
     });
-});
\ No newline at end of file
+
+    // Shows set priority modal
+    $(".check-priority").click(function(e) {
+        var $this = $(this);
+        var a = e.target;
+        var priority = a.getAttribute("data-priority");
+        var emails = a.getAttribute("data-escal-emails");
+        console.log(priority, emails)
+
+        $("#escalation-emails_tag").val(emails);
+        $("#check-priority").val(priority);
+        if($("#check-priority").val() === 1){
+            console.log("val = 1")
+           $("#check-priority-toggle").bootstrapToggle('High');
+        }else{
+            console.log("val = 0")
+           $("#check-priority-toggle").bootstrapToggle('Low');
+        }
+        $("#set-priority-modal").modal({ "show": true, "backdrop": "static" });
+        return false;
+    });
+
+    // Sets priority slider
+    var prioritySlider = document.getElementById("priority-slider");
+    noUiSlider.create(prioritySlider, {
+        start: [0],
+        connect: 'lower',
```

Comment:
Be consistent with the type of quotation marks you choose to use.

Reply:
Done.